docs/content/reference/versioning-support.md

```diff
@@ -11,18 +11,18 @@ There are different components that might require independent versioning and sup
 
 ### HO
 
-- A HO is intended to be released for each OCP release.
-- A HO released for OCP N minor release must support N, N-1 and N-2 OCP minor releases.
-- A HO release if anything, will only do best effort to support future OCP versions.
+- A HO is intended to be released for each OCP minor release.
+- A HO released for OCP N minor release must support guest clusters of N, N-1 and N-2 OCP minor releases.
+- A HO must be updated before future OCP minor release can be deployed or updated. For example, a HO released for OCP N minor release does not support future guest cluster N+1 minor releases. A HO must be updated to match the N+1 release before guest clusters can be deployed or updated to N+1.
```

Comment:
This whole thing is ill-defined in my mind as there is no "HO released for OCP N minor release" as the HO is not versioned to a particular OCP release.  It can _support_ particular OCP releases.

Reply:
The idea of an HO release is also fuzzy as we do not version it and only the HO built from `main` should ever be used.

Reply:
> This whole thing is ill-defined in my mind as there is no "HO released for OCP N minor release" as the HO is not versioned to a particular OCP release. It can support particular OCP releases.
> The idea of an HO release is also fuzzy as we do not version it and only the HO built from main should ever be used.

For the self managed use case, I'm not sure saying the HO doesn't have a corresponding OCP minor release is accurate.

The hypershift repo is versioned and aligned to OCP releases. We have HOrelease branches that directly correlate to OCP releases. Those HO release branches then directly map to MCE releases as well which also align to the OCP release schedule.

For example

Hypershift operator in release branch release-4.12 maps directly to (and is automatically synced with) MCE 2.2.z releases. MCE 2.2 is the stable version of MCE that is installed in 4.12 OCP clusters.

So, HO in branch release-4.12 is guaranteed to work with guest clusters <= 4.12. If someone wants to transition their guest clusters to 4.13, the HO needs to be updated to something derived from the hypershift release-4.13 branch. In the self managed use case, that's MCE 2.3.

That relationship of the HO release branch to supported guest cluster versions is what I'm trying to communicate here.